### DISCUSSION TITLE
PAC admin delete command failing with 500 internal server error

### DISCUSSION THREAD

**Discussion by samuthrakumar** [microsoft/powerplatform-build-tools#604] [Q&A]
I am running the pac admin delete command for an environment, which constantly fails with 500. Does anyone know how I use the correlation ID and tracking ID to find more details about the failure? Error: -HttpStatusCode":500,"RequestMethod":"GET","RequestUri":"
[xxxx","CorrelationIdName":"x-ms-correlation-id","CorrelationId":"**redacted**","ResponseBody":"{\\"message\\":\\"Encountered]()
internal server error. The tracking Id is '**redacted**'.\\",\\"operations\\":

**Comment by petrochuk** [comment]
You can contact support and they will look into this specific case:
https://learn.microsoft.com/en-us/power-platform/admin/get-help-support

**Reply by samuthrakumar** [reply]
Thanks, I will reach out to the support.
